tools/SourceKit/include/SourceKit/Core/LangSupport.h

```diff
@@ -993,6 +1004,12 @@ class LangSupport {
       std::function<void(const RequestResult<RelatedIdentsInfo> &)>
           Receiver) = 0;
 
+  virtual void findInactiveRangesInFile(
+      StringRef Filename, bool CancelOnSubsequentRequest,
+      ArrayRef<const char *> Args, SourceKitCancellationToken CancellationToken,
```

Comment:
The `CancelOnSubsequentRequest` only exists in the cursor info request because it implicitly cancelled old requests if new requests came in before we had explicit cancellation in sourcekitd. Since this is a new request, I think we don’t need `CancelOnSubsequentRequest` and instead clients should explicitly cancel the request if they are no longer interested in the result.